composition-js/src/__tests__/override.compose.test.ts

```diff
@@ -0,0 +1,270 @@
+import { printSchema } from "@apollo/federation-internals";
+import gql from "graphql-tag";
+import "./matchers";
+import {
+  assertCompositionSuccess,
+  schemas,
+  errors,
+  composeAsFed2Subgraphs,
+} from "./compose.test";
+
+describe("composition involving @override directive", () => {
+  it("@override whole type", () => {
```

Comment:
That test is the "idealised" case, but it would be nice to add tests for `@override` on type but where there is some mismatch between the fields, just to make sure this works as expected. Something like:
```graphql
# Subgraph1
type T @key(fields: "k") @override(from: "Subgraph2") {
  k: ID
  a: Int
  b: Int
}
```
```graphql
# Subgraph1
type T @key(fields: "k") {
  k: ID
  b: Int
  c: Int
}

```